src/sdkClient/sdkClient.ts

```diff
@@ -5,12 +5,37 @@ import { clientFactory } from './client';
 import { clientInputValidationDecorator } from './clientInputValidation';
 import { ISdkFactoryContext } from '../sdkFactory/types';
 
+const COOLDOWN_TIME = 3600;
```

Comment:
Emma, please rename it to `COOLDOWN_TIME_IN_MILLIS` or `COOLDOWN_TIME_IN_SECONDS`, depending the case, to not confuse the value.
Is it 3.6 seconds (3600 millis) ? is that correct?